sqlscripts/local/0003-jaeger-operations.sql

```diff
@@ -8,6 +8,7 @@ AS SELECT
     toDate(timestamp) AS date,
     service,
     operation,
-    count() as count
+    count() as count,
+    if(has(tags.key, 'span.kind'), tags.value[indexOf(tags.key, 'span.kind')], '') as spankind
```

Comment:
could you please explain this one? Why are we treating span.kind differently than other tags?

Reply:
Span.kind tag is used to define SpanKind field in spanstore.Operation. Spankind column is used to get it easier in GetOperations.